src/main/java/rx/Observable.java

```diff
@@ -20,6 +20,7 @@
 import rx.functions.*;
 import rx.internal.operators.*;
 import rx.internal.producers.SingleProducer;
+import rx.internal.operators.OperatorOnBackpressureBuffer.OnOverflow;
```

Comment:
Makes sense.

Any preference rather than in Observable.java? This seems to be the first case of an enum that's part of the public API (other than Notification.Kind, which is in Notification itself).